src/tests/ftest/security/pool_security_acl.py

```diff
@@ -0,0 +1,403 @@
+            add_del_user(self.hostlist_clients, "useradd", username)
+            user_list.append(new_user)
+        for gid in range(num_group):
+            groupname = user_prefix + "_testGrp_" + str(gid + 1)
+            new_group = "A:G:" + groupname + "@:" + PERMISSIONS[(gid + 2) % 4]
+            add_del_user(self.hostlist_clients, "groupadd", groupname)
+            group_list.append(new_group)
+        permission_list = group_list + user_list + current_user_acl
+        random.shuffle(permission_list)
+        with open(acl_file, "w") as test_file:
+            test_file.write("\n".join(permission_list))
+        return permission_list
+
+    def cleanup_user_group(self, num_user, num_group):
+        '''
+        Deascription:
+            Remove the daos acl user and group on host.
+        Args:
+            num_user: number of user to be cleaned.
+            num_group: number of group name to be cleaned.
+        Return:
+            none.
+        '''
+        for user in range(num_user):
+            username = "daos_ci_tester_" + str(user + 1)
+            add_del_user(self.hostlist_clients, "userdel", username)
+        for group in range(num_group):
+            groupname = "daos_ci_testGrp_" + str(group + 1)
+            add_del_user(self.hostlist_clients, "groupdel", groupname)
+
+    def pool_acl_verification(self, current_user_acl, read, write):
+        '''
+        Deascription:
+            Daos pool security verification with acl file.
+            Steps:
+                (1)Setup dmg tool for creating a pool
+                (2)Generate acl file with permissions
+                (3)Create a pool with acl
+                (4)Verify the pool create status
+                (5)Get the pool's acl list
+                (6)Verify pool read operation
+                (7)Verify pool write operation
+                (8)Cleanup user and destroy pool
+        Args:
+            current_user_acl: acl with read write access credential.
+            read: expecting read permission.
+            write: expecting write permission.
+        Return:
+            pass to continue.
+            fail to report the testlog and stop.
+        '''
+
+        # (1)Create daos_shell command
+        dmg = DmgCommand(os.path.join(self.prefix, "bin"))
+        dmg.get_params(self)
+        port = self.params.get("port", "/run/server_config/*", 10001)
+        get_acl_file = self.params.get("acl_file", "/run/pool_acl/*",
+                                       "acl_test.txt")
+        acl_file = os.path.join(self.tmp, get_acl_file)
+        num_user = self.params.get("num_user", "/run/pool_acl/*")
+        num_group = self.params.get("num_group", "/run/pool_acl/*")
+        servers_with_ports = [
+            "{}:{}".format(host, port) for host in self.hostlist_servers]
+        dmg.hostlist.update(",".join(servers_with_ports), "dmg.hostlist")
+        self.log.info("  (1)dmg= {}".format(dmg))
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments